ui/guest/src/contentController.ts

```diff
@@ -787,7 +787,9 @@ fromTopic(fetchGuestModelComplete.type).subscribe((action: StandardAction<FetchG
       mhm[id].parentContainerFieldPath = mhm[id].parentContainerFieldPath ?? hierarchyMap[id].parentContainerFieldPath;
       mhm[id].parentContainerFieldIndex =
         mhm[id].parentContainerFieldIndex ?? hierarchyMap[id].parentContainerFieldIndex;
-      mhm[id].children = mhm[id].children ?? hierarchyMap[id].children;
+      // Flipping around from the fix on ticket #6433 for ticket #8154. The children coming from Host must be the latest.Add commentMore actions
+      // TODO: Observed the local hierarchyMap[id].children containing duplicates. Where is that coming from? Are children computed locally?
```

Comment:
@rart a couple of questions:
- In the previous comment at the end, it says 'Add commentMore actions'. What's the idea there?
- Any particular scenario where you saw the duplicate children? I wasn't able to see duplicates while duplicating and or removing content

Reply:
- Ignore the "add commentmore actions" looks like a typo. Don't carry that piece of the comment.
- It was seeing in a specific site. Not to worry at this time.